File: dollarx/src/main/java/info/dollarx/custommatchers/hamcrest/HasElements.java
package info.dollarx.custommatchers.hamcrest;

import info.dollarx.Path;

public class HasElements {
    private final Path path;

    public HasElements(final Path path){
        this.path = path;
    }

    public HasElementNTimes present(int nTimes) {
        return new HasElementNTimes(path, nTimes);
    }
}

File: dollarx/src/main/java/com/github/loyada/dollarx/custommatchers/hamcrest/HasElements.scala
package com.github.loyada.dollarx.custommatchers.hamcrest

import com.github.loyada.dollarx.Path

case class HasElements(path: Path) {
  def present(nTimes: Int): HasElementNTimes = {
     new HasElementNTimes(path, nTimes)
  }
}
